config.md

```diff
@@ -34,6 +34,25 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 }
 ```
 
+## Mount Points
+
+You can add array of mount points inside container. Each record in this array
+must have configuration in runtime config in *mounts object*, where keys are
+*name* of mount point.
+
+* **name** (string, required) Name of mount point. Used for configuration lookup
+in runtime config.
+* **path** (string, required) Destination of mount point: path inside container.
+
+*Example*
+
+```json
+"mounts": {
+    "name": "proc",
+    "path": "/proc",
+}
```

Comment:
If we want to list several mount points in the `runtime.json` docs, I think we might as well map all of those here.  If we like the abbreviated example here, I think we might as well abbreviate the `config.json` example to match ;).  So I don't mind which way we go, but I think the examples should fit together.